lib/fauna/query.rb

```diff
@@ -300,6 +311,17 @@ def self.divide(*numbers)
 
   private
 
+    # Queries that accept lambdas can take either a manually construced lambda or a block.
+    def self.to_lambda(lambda_expr, lambda_block)
```

Comment:
I guess. This will drop a block if it is provided along with lambda_expr, but that probably won't be a problem.